<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>河钢乐亭薪酬管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../conf/ICON.ico</normaloff>../conf/ICON.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>工资核算</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,8,4">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>第一步：日期确定</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1,0,0,0,1,2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>请输入工资核算日期：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_year">
             <property name="minimum">
              <number>2019</number>
             </property>
             <property name="maximum">
              <number>2020</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>年</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_month">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>12</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>月</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>第二步：数据导入</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="3">
              <widget class="QPushButton" name="btn_table_house_insurance">
               <property name="text">
                <string>公积金缴费明细</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="btn_table_endowment_insurance_roster">
               <property name="text">
                <string>养老保险花名册</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QPushButton" name="btn_table_people_info">
               <property name="text">
                <string>人员信息表</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="btn_table_tax_extra">
               <property name="text">
                <string>财务附加计税明细</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="btn_table_endowment_insurance_detail">
               <property name="text">
                <string>养老保险缴费明细</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="btn_table_position_change">
               <property name="text">
                <string>岗位变更表</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="btn_table_sap_workday">
               <property name="text">
                <string>sap考勤表</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QPushButton" name="btn_table_unemployment_insurance_detail">
               <property name="text">
                <string>失业保险缴费明细</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="btn_table_standard">
               <property name="text">
                <string>工资标准表</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QPushButton" name="btn_table_insurance_extra_payment">
               <property name="text">
                <string>保险补缴明细</string>
               </property>
              </widget>
             </item>
             <item row="4" column="3">
              <widget class="QPushButton" name="btn_table_extra_workincome">
               <property name="text">
                <string>工资补发补扣</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QPushButton" name="btn_table_last_month_tax_detail">
               <property name="text">
                <string>上月缴税明细</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="btn_table_medical_insurance_detail">
               <property name="text">
                <string>医疗保险缴费明细</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QPushButton" name="btn_table_civil_servants_insurance_detail">
               <property name="text">
                <string>公务员医疗缴费明细</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="btn_table_tax_remove">
               <property name="text">
                <string>个人附加扣除</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="btn_table_medical_insurance_roster">
               <property name="text">
                <string>医疗保险缴费花名册</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QPushButton" name="btn_last_month_wage_detail">
               <property name="text">
                <string>上月工资明细表</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>第三步：工资生成</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>财务</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="btn_finace_new_info">
                <property name="text">
                 <string>新增人员信息</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_finace_tax_calc_template">
                <property name="text">
                 <string>正常薪金所得（算税模板）</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>人力</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QPushButton" name="btn_workincome_calc">
                  <property name="text">
                   <string>工资计算</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="btn_workincome_summary">
                  <property name="text">
                   <string>工资汇总表</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QPushButton" name="btn_wage_split">
                  <property name="text">
                   <string>工资拆分表</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="btn_workincome_detail">
                  <property name="text">
                   <string>工资明细表</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>奖金核算</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,5,5">
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>第一步：日期确定</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,1,0,0,0,1,2">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>请输入奖金核算日期：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_bouns_year">
             <property name="minimum">
              <number>2019</number>
             </property>
             <property name="maximum">
              <number>2020</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>年</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_bouns_month">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>12</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>月</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>第二部：数据导入</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_sum_excel_to_sql">
             <property name="text">
              <string>本月各部门绩效汇总</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_bouns_extra">
             <property name="text">
              <string>奖金补发补扣</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>第三步：奖金生成</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
           <item>
            <widget class="QGroupBox" name="groupBox_9">
             <property name="title">
              <string>财务</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btn_finace_tax_calc_template_2">
                <property name="text">
                 <string>正常薪金所得（算税模板）</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_10">
             <property name="title">
              <string>人力</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="1">
                 <widget class="QPushButton" name="btn_workincome_summary_2">
                  <property name="text">
                   <string>奖金汇总表</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="btn_bouns_detail">
                  <property name="text">
                   <string>奖金明细表</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="btn_bouns_calc">
                  <property name="text">
                   <string>奖金计算</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QPushButton" name="btn_wage_split_2">
                  <property name="text">
                   <string>奖金拆分表</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="title">
        <string>报表生成</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>下月预算表</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_11">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>828</width>
          <height>78</height>
         </rect>
        </property>
        <property name="title">
         <string>第一步：日期确定</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,1,0,0,0,1,2">
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>请输入工资核算日期：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_year_2">
           <property name="minimum">
            <number>2019</number>
           </property>
           <property name="maximum">
            <number>2020</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>年</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_month_2">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>月</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab4">
       <attribute name="title">
        <string>数据分析</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
